classes/Attribute.php

```diff
@@ -339,13 +339,13 @@ public function updatePosition($way, $position)
      * @param bool $use_last_attribute
      * @return bool $return
      */
-    public function cleanPositions($id_attribute_group, $use_last_attribute = true)
+    public static function cleanPositions($id, $id_attribute_group, $use_last_attribute = true)
```

Comment:
We can't do this now, 1.7 has been released so now, if we do this, retro-compatibility will be broken :/